nerdctl/Tiltfile

```diff
@@ -0,0 +1,107 @@
+def _quote(v):
+    if os.name == 'nt':
+        return v
+    return shlex.quote(v)
+
+def _env_key(ref):
+    out = 'NERDCTL_DF_'
+    for c in ref.elems():
+        if c.isalnum():
+            out += c
+        else:
+            out += '_'
+    return out
+
+def nerdctl_build(
+    ref,
+    context,
+    build_args={},
+    dockerfile=None,
+    dockerfile_contents=None,
+    live_update=[],
+    match_in_env_vars=False,
+    ignore=[],
+    entrypoint=[],
+    target='',
+    ssh='',
+    secret='',
+    extra_tag='',
+    cache_from=[],
+    platform='',
+):
+    cmd = ['nerdctl']
+
+    cmd += ['--namespace', 'k8s.io']
+
+    cmd += ['build']
+
+    for arg, value in build_args.items():
+        cmd += ['--build-arg', arg + '=' + value]
+
+    if dockerfile:
+        cmd += ['-f', dockerfile]
+    elif dockerfile_contents:
+        cmd += ['-f', '-']
+
+    if target:
+        cmd += ['--target', target]
+
+    if ssh:
+        cmd += ['--ssh', ssh]
+
+    if secret:
+        cmd += ['--secret', secret]
+
+    if extra_tag:
+        if type(extra_tag) == 'string':
+            cmd += ['-t', extra_tag]
+        else:
+            for t in extra_tag:
+                cmd += ['-t', t]
+
+    if cache_from:
+        if type(cache_from) == 'string':
+            cmd += ['--cache-from', cache_from]
+        else:
+            for v in cache_from:
+                cmd += ['--cache-from', v]
+
+    # add the expected ref arg _after_ quoting since we actually
+    # want shell expansion here and quoting will turn this into
+    # a literal
```

Comment:
wanna add some quoting above this comment? :)

Reply:
😳  (fixed in 62b3032)